DEVELOPMENT.md

```diff
@@ -464,7 +464,7 @@ This will compile binaries for all supported platforms, `deb`, `rpm` and Snapcra
 
 Releasing a new version consists of the following steps:
 
-1. Creating a `release/<version>` branch(further, called "release branch") (e.g. `release/3.2.1`).
+1. Creating a `release/<version>` branch off `master` or `backports/<minor>` (further, called "release branch") (e.g. `release/v3.2.1`).
```

Comment:
I thought you wanted `backport/<minor>` since we also have singular `release/<version>`

Reply:
I don't care really. With `3.6.2` I chose for singular, not knowing there's `backports/*` branch protection setup

Reply:
I think we should keep them consistent. So if we do singular `release/<version>`, we should also do singular `backport/<minor>`.